php/WP_CLI/FileCache.php

```diff
@@ -314,6 +314,9 @@ public function prune() {
 	 */
 	protected function ensure_dir_exists( $dir ) {
 		if ( ! is_dir( $dir ) ) {
+			if ( preg_match( '{(^|[\\\\/])(\$null|nul|NUL|/dev/null)([\\\\/]|$)}', $dir ) ) {
+				return false;
+			}
```

Comment:
CS: Leave an empty line between two pieces of logic:
```suggestion
			}

```